pyproject.toml

```diff
@@ -192,12 +188,9 @@ dependencies = [
   "grpc-interceptor[testing]",
   "responses",
   "tiktoken",
-  "typing-extensions==4.5.0; python_version=='3.8'",
-  "typing-extensions==4.6.0; python_version=='3.9'",
```

Comment:
Had to unpin this one due to the following error. Incidentally, Colab is already on 4.12.2, so we no longer need to pin this for colab.
```
src/phoenix/server/app.py:31: in <module>
    import strawberry
../Library/Application Support/hatch/env/virtual/arize-phoenix/a0fu6yA-/test.py3.9/lib/python3.9/site-packages/strawberry/__init__.py:7: in <module>
    from . import experimental, federation, relay
../Library/Application Support/hatch/env/virtual/arize-phoenix/a0fu6yA-/test.py3.9/lib/python3.9/site-packages/strawberry/experimental/__init__.py:2: in <module>
    from . import pydantic
../Library/Application Support/hatch/env/virtual/arize-phoenix/a0fu6yA-/test.py3.9/lib/python3.9/site-packages/strawberry/experimental/pydantic/__init__.py:1: in <module>
    from .error_type import error_type
../Library/Application Support/hatch/env/virtual/arize-phoenix/a0fu6yA-/test.py3.9/lib/python3.9/site-packages/strawberry/experimental/pydantic/error_type.py:24: in <module>
    from strawberry.experimental.pydantic.utils import (
../Library/Application Support/hatch/env/virtual/arize-phoenix/a0fu6yA-/test.py3.9/lib/python3.9/site-packages/strawberry/experimental/pydantic/utils.py:28: in <module>
    from strawberry.types.private import is_private
../Library/Application Support/hatch/env/virtual/arize-phoenix/a0fu6yA-/test.py3.9/lib/python3.9/site-packages/strawberry/types/__init__.py:2: in <module>
    from .execution import ExecutionContext, ExecutionResult, SubscriptionExecutionResult
../Library/Application Support/hatch/env/virtual/arize-phoenix/a0fu6yA-/test.py3.9/lib/python3.9/site-packages/strawberry/types/execution.py:116: in <module>
    class SubscriptionExecutionResult(Protocol):
/opt/homebrew/Cellar/python@3.9/3.9.18_2/Frameworks/Python.framework/Versions/3.9/lib/python3.9/typing.py:1369: in runtime_checkable
    raise TypeError('@runtime_checkable can be only applied to protocol classes,'
E   TypeError: @runtime_checkable can be only applied to protocol classes, got <class 'strawberry.types.execution.SubscriptionExecutionResult'>
```